capa/render/verbose.py

```diff
@@ -71,6 +71,20 @@ def render_meta(ostream, doc):
             + sum(doc["meta"]["analysis"]["feature_counts"]["functions"].values()),
         ),
     ]
+
+    # when there are multiple rule paths,
+    # display each one on its own line, like:
+    #
+    #     base address            0x10000000
+    #     rules                   /path/1
+    #                             /path/2
+    #     function count          2
+    if len(doc["meta"]["analysis"]["rules"]) > 1:
+        idx = rows.index(("rules", doc["meta"]["analysis"]["rules"][0])) + 1
+        for rule in doc["meta"]["analysis"]["rules"][1:]:
+            rows.insert(idx, ("", rule))
+            idx += 1
```

Comment:
"\n".join(...) should also work?

```
print(tabulate.tabulate([("left column", "right 1\nright 2")]))
-----------  -------
left column  right 1
             right 2
-----------  -------
```

Reply:
whoa, thats a lot cleaner, thanks!

Reply:
fixed in fccca823c5746cebedea240e3833bc0ee72d988d

Reply:
oh yeah!